custom_components/mail_and_packages/const.py

```diff
@@ -275,11 +277,20 @@
         "subject": ["Hermes has successfully delivered your"],
     },
     "hermes_delivering": {
-        "email": ["donotreply@myhermes.co.uk"],
-        "subject": ["parcel is now with your local Hermes courier"],
+        "email": [
+            "donotreply@myhermes.co.uk",
+            "noreply@paketankuendigung.myhermes.de",
+        ],
+        "subject": [
+            "parcel is now with your local Hermes courier",
+            "Ihre Hermes Sendung",
+        ],
+        "body": [
+            "Voraussichtliche Zustellung",
+        ],
     },
     "hermes_packages": {},
-    "hermes_tracking": {"pattern": ["\\d{16}"]},
+    "hermes_tracking": {"pattern": ["\\d{11,16}"]},
```

Comment:
Are you sure the tracking number can be 11 or 16 digits?

Reply:
According to https://www.myhermes.de/empfangen/sendungsverfolgung/

Between 11 and 20 digits